twitter-lm.html

```diff
@@ -0,0 +1,571 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="twitter-lm.css" />
+    <script
+    src="https://kit.fontawesome.com/f9c0780752.js"
+    crossorigin="anonymous"></script>
+    <title>Twitter Clone</title>
+  </head>
+  <body>
+
+    <div class="container-wrapper">
+      <div class="container three-column-grid__expand-two center-wrapper-md">
+
+        <aside class="left-side">
+          <div class="logo">
+            <i class="fa-brands fa-twitter"></i>
+          </div>
+          <ul class="list-container text-heading">
+            <li class="list-item">
+              <a class="twitter-links" href="https://www.twitter.com/home" target="_blank">Home</a>
+            </li>
+            <li class="list-item">
+              <div class="icon-wrapper">
+                <i class="fa-regular fa-hashtag"></i>
+              </div>
+              <a class="twitter-links2" href="https://www.twitter.com/explore" target="_blank">Explore</a>
+            </li>
+            <li class="list-item">
+              <div class="icon-wrapper">
+                <i class="fa-regular fa-bell"></i>
+            </div>
+            <a class="twitter-links2" href="https://www.twitter.com/notifications" target="_blank">Notifications</a>
```

Comment:
Suggest you review HTML formatting. It's not immediately obvious that the `<a>` tag is inside the `<li>`. 

Have a look at using Prettier extension in VSCode to automatically format your html files on save or paste?